storybook/src/components/PageMenu/PageMenu.stories.tsx

```diff
@@ -17,54 +17,49 @@ export default meta
 type Story = StoryObj<typeof meta>
 
 export const Default: Story = {
-  render: () => (
-    <PageMenu>
-      <PageMenu.Link href="#">English</PageMenu.Link>
-      <PageMenu.Link href="#" icon={LoginIcon}>
+  args: {
+    children: [
+      <PageMenu.Link href="#" key={1}>
+        English
+      </PageMenu.Link>,
+      <PageMenu.Link href="#" icon={LoginIcon} key={2}>
         Inloggen Mijn Amsterdam
-      </PageMenu.Link>
-    </PageMenu>
-  ),
-  parameters: {
-    docs: {
-      source: { type: 'dynamic' },
-    },
+      </PageMenu.Link>,
+    ],
   },
 }
 
 export const AlignEnd: Story = {
   args: {
     alignEnd: true,
-  },
-  render: (args) => (
-    <PageMenu {...args}>
-      <PageMenu.Link href="#" key="onderzoeken">
+    children: [
+      <PageMenu.Link href="#" key={1}>
         Onderzoeken
-      </PageMenu.Link>
-      <PageMenu.Link href="#" key="bezoeken">
+      </PageMenu.Link>,
+      <PageMenu.Link href="#" key={2}>
         Bezoeken
-      </PageMenu.Link>
-      <PageMenu.Link href="#" key="archiveren">
+      </PageMenu.Link>,
+      <PageMenu.Link href="#" key={3}>
```

Comment:
I understand how numbers seem simple and nice, but keys prevent having to change other values when inserting items into a list.

Reply:
I'm fine either way, as long as we do it consistently. Sometimes using a string for a key and mostly using numbers implies that the string has meaning imo, which it hasn't. Always using a string would fix that, but it'd be a lot more code.

I think changing everything to strings is more work then we'd save when we have to insert an item into a list once or twice, so I'm leaning towards numbers.

Reply:
Fair enough, I can go along with numbers.